ivy/array/manipulation.py

```diff
@@ -351,6 +351,13 @@ def constant_pad(
         ivy.Array instance method variant of ivy.constant_pad. This method simply
         wraps the function, and so the docstring for ivy.constant_pad also applies
         to this method with minimal changes.
+
+        Examples
+        --------
+        >>> x = ivy.array([-4.7, -2.3, 0.7]) #x.shape->(3,)
+        >>> y = x.expand_dims() #y.shape->(1, 3)
```

Comment:
it seems like you copied over an example from a different function

Reply:
Huge mistakes. I'd be on it shortly